release/ray_release/byod/build.py

```diff
@@ -244,36 +244,13 @@ def _download_dataplane_build_file() -> None:
         assert digest == DATAPLANE_DIGEST, "Mismatched dataplane digest found!"
 
 
-def _ray_image_exist(ray_image: str) -> bool:
+def _image_exist(image: str) -> bool:
     """
     Checks if the given image exists in Docker
     """
     p = subprocess.run(
-        ["docker", "manifest", "inspect", ray_image],
+        ["docker", "manifest", "inspect", image],
         stdout=sys.stderr,
         stderr=sys.stderr,
     )
     return p.returncode == 0
-
-
-def _byod_image_exist(test: Test, base_image: bool = True) -> bool:
-    """
-    Checks if the given Anyscale BYOD image exists.
-    """
-    if os.environ.get("BYOD_NO_CACHE", False):
```

Comment:
Removing this env-var while I'm here, the image is now built within the pipeline so there are no use cases for this